test/remote/account_preverified_token_tests.js

```diff
@@ -1,131 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-'use strict'
-
-const assert = require('insist')
-var TestServer = require('../test_server')
-const Client = require('../client')()
-var JWTool = require('fxa-jwtool')
-
-var config = require('../../config').getProperties()
-var secretKey = JWTool.JWK.fromFile(
-  config.secretKeyFile,
-  {
-    jku: config.publicUrl + '/.well-known/public-keys',
-    kid: 'dev-1'
-  }
-)
-
-function fail() { throw new Error('call succeeded when it should have failed')}
-
-function nowSeconds() {
-  return Math.floor(Date.now() / 1000)
-}
-
-describe('remote account preverified token', function() {
-  this.timeout(15000)
-  let server
-  before(() => {
-    process.env.TRUSTED_JKUS = 'http://127.0.0.1:9000/.well-known/public-keys'
-    process.env.IP_PROFILING_ENABLED = true
```

Comment:
My theory is, this line used to execute before the failing tests, and it meant all subsequent tests could skip sign-in confirmation.  With it removed, the failing tests are now trying to access keys without verifying their signin, producing the "unverified account" errors.